src/mscorlib/System.Private.CoreLib.csproj

```diff
@@ -385,6 +383,8 @@
   <ItemGroup>
     <Compile Include="$(BclSourcesRoot)\System\Reflection\Assembly.CoreCLR.cs" />
     <Compile Include="$(BclSourcesRoot)\System\Reflection\AssemblyName.cs" />
+    <Compile Include="$(BclSourcesRoot)\System\Reflection\AssemblyNameHelpers.cs" />
```

Comment:
Note the files that are shared in coreclr are listed in \mscorlib\shared\System.Private.CoreLib.Shared.shproj not in thecproj.